### PR TITLE
add domain

### PR DESCRIPTION
**Pull request by yoeldesdev**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/yoeldesdev.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ "description": "yoeldesdev web",
```
Comment (21Z) [inline domains/yoeldesdev.json:1]: Invalid JSON
```suggestion
{
 "description": "yoeldesdev web",
```

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/yoeldesdev.json (hunk 1 above)
Comment (Stef-00012) [reply domains/yoeldesdev.json:1]: accept @21Z's suggestion

File: domains/yoeldesdev.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    "repo": "https://github.com/yoeldesdev/yoeldesdev.github.io>",
+    "owner": {
+        "username": "yoeldesdev",
+        "email": "yoeldesdev@gmail.com",
```
Comment (Stef-00012) [inline domains/yoeldesdev.json:5]: remove the `,` here

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/yoeldesdev.json (hunk 2 above)
Comment (yoeldesdev) [reply domains/yoeldesdev.json:6]: ok

**Review by Real-Yash — APPROVED** [review]

File: domains/yoeldesdev.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+ "description": "yoeldesdev web",
+    "repo": "https://github.com/yoeldesdev/yoeldesdev.github.io>",
+    "owner": {
+        "username": "yoeldesdev",
+        "email": "yoeldesdev@gmail.com",
```
Comment (CuteDog5695) [inline domains/yoeldesdev.json:6]:
```suggestion
        "email": "yoeldesdev@gmail.com"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**